backend/schema/corpora_schema.md

```diff
@@ -296,8 +205,8 @@ X_umap for example.
 
 Finally, the dataset-level metadata is stored in `uns`, which is just key-value pairs.
 
-Note that anndata supports "layers" and "raw" values for counts. Those are permitted, but Corpora will treat `X` as the "final" matrix for further analysis
-and visualization. Once anndata [unifies its treatment of layers](https://github.com/theislab/anndata/issues/244), Corpora will use the "default" as the
+Note that anndata supports "layers" and "raw" values for counts. Those are permitted, but cellxgene will treat `X` as the "final" matrix for further analysis
```

Comment:
This doc mentions "data integration" & "further analysis" and includes some details on how to enable it, but not much detail on what the looks like. Can that use case be expanded?

Reply:
Well "data integration" is somewhat left as an exercise for the reader. The idea is that somebody will find the data corpus useful for integration. Here though, I think the term "further analysis" could be clarified. It really just means that cellxgene displays the "X" matrix.

Reply:
So if the reader embarks on data integration, they are free to download the matrix and thus, use whichever layer they please, right?

Reply:
Yeah

Reply:
I would suggest removing the assumption about what will be used for further analysis to ...
> Those are permitted, but cellxgene will treat X as the "final" matrix for visualization

.